api/seeds/shared/apds.js

```diff
@@ -1,10 +0,0 @@
-const { states } = require('../../util/states');
-
-const apdEntries = states.map(state => ({
-  status: 'draft',
-  state_id: state.id
-}));
-
-exports.seed = async knex => {
-  await knex('apds').insert(apdEntries);
-};
```

Comment:
Now that we have a way to make APDs, no need to seed them.

Reply:
👍